src/lib/routes/admin-api/state.ts

```diff
@@ -102,6 +102,7 @@ class StateController extends Controller {
         const userName = extractUsername(req);
         const { drop, keep } = req.query;
         // TODO: Should override request type so file is a type on request
+        // biome-ignore lint/suspicious/noImplicitAnyLet: <explanation>
```

Comment:
This is prod code, maybe we can add an explanation, or just don't know why this is ignored rather than <explanation>